platforms/chibios/drivers/i2c_master.c

```diff
@@ -90,22 +102,30 @@
 
 static uint8_t i2c_address;
 
+#if defined(MCU_RP)
+#    if !defined(I2C1_CLOCK_SPEED)
+#        define I2C1_CLOCK_SPEED 400000
+#    endif
+
+static const I2CConfig i2cconfig = {.baudrate = I2C1_CLOCK_SPEED};
+#else
```

Comment:
This should follow the existing convention below rather than ifdef the existing code. Better would be to try and confirm to USE_I2CV1_CONTRIB?

eg
```c
#elif defined(MCU_RP)
    .baudrate = I2C1_CLOCK_SPEED
#elif defined(WB32F3G71xx)
```

Reply:
Done! All i2c driver adjustments are now handled via `chibios_config.h`